pkg/operator/controller/machineconfig/controller.go

```diff
@@ -343,11 +344,11 @@ func (r *MachineConfigReconciler) ensureProfConfDisabled(ctx context.Context) (b
 		return true, r.revertNodeUnlabeling(ctx)
 	}
 
-	// FIXME: shouldn't these conditions be always set to False, when ensureProfConfDisabled() is called?
```

Comment:
Yes, but DebugReady should be set to InProgress when there is a change and requires update.